sample-apps/javascript-sample-app/request-metrics.js

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS'" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+const TOTAL_BYTES_SENT_METRIC = 'totalBytesSent';
+const TOTAL_API_REQUESTS = 'apiRequests';
+const LATENCY_TIME = 'latencyTime';
+const meter = require('./meter');
+
+// variable to track number of api requests.
+let n = 0;
```

Comment:
Can we use a better variable name here instead of n? It looks to have a purpose so we should assign a variable name that represents that purpose.

Reply:
Thanks. Updated.